frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts

```diff
@@ -44,3 +45,6 @@ export const infraProvisionerMap = {
   aws: 'kubernetes.io/aws-ebs',
   vsphere: 'kubernetes.io/vsphere-volume',
 };
+
+export const MIN_NODE_MEMORY = convertToBaseValue('64 Gi');
+export const MIN_NODE_CPU = 16;
```

Comment:
I still have the same concerns that we've created constants, but the same values are in messages in a different file. These can easily get out of sync.